src/main/java/ch/admin/bag/covidcertificate/web/controller/RevocationController.java

```diff
@@ -162,10 +160,11 @@ private void logRevocationCheckKpi(String userExtId, SystemSource systemSource)
         Jwt token = jeapAuthorization.getJeapAuthenticationToken().getToken();
         String relevantUserExtId = UserExtIdHelper.extractUserExtId(token, userExtId, systemSource);
         LocalDateTime kpiTimestamp = LocalDateTime.now();
+        // TODO: check with Splunk-Reports
         log.info("kpi: {} {} {}",
                 kv(KPI_TIMESTAMP_KEY, kpiTimestamp.format(LOG_FORMAT)),
                 kv(KPI_TYPE_KEY, KPI_TYPE_MASS_REVOCATION_CHECK),
-                kv(PREFERRED_USERNAME_CLAIM_KEY, relevantUserExtId));
+                kv(KPI_UUID_KEY, relevantUserExtId));
```

Comment:
Muss hier allenfalls in der KPI DB noch korrigiert werden?

Reply:
Nein, die KPI Tabelle ist korrekt befüllt. Wird ja separat gemacht